smdebug/tensorflow/keras.py

```diff
@@ -366,6 +385,90 @@ def _add_metric(self, metric_name, metric_value: tf.Tensor = None):
             coll.set_tensor_ref(TensorRef.from_non_graph_var(metric_name))
         self.tensor_to_collections[metric_name] = {coll}
 
+    def save_custom_tensor(self, tensor_name, tensor_value, collections_to_write):
```

Comment:
so same tensor can be saved under multiple collections? and we are relying on the user to pass the right arguments?

could you add the doc stub for this?

is this applicable only to TF?

if this is a user facing API, you will have to update docs.